[FW][FIX] sipreco_subsidy_management:expedient creation restriction

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/odoo-public-administration#537

mav-adhoc committed:
[FIX] sipreco_subsidy_management:expedient creation restriction

X-original-commit: 898c678b114fef5268a3562613de37a30459b3e3

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-public-administration/pull/538).

fw-bot-adhoc commented:
@mav-adhoc @jcadhoc cherrypicking of pull request ingadhoc/odoo-public-administration#537 failed.

stdout:
```
CONFLICT (rename/delete): sipreco_subsidy_management/wizards/create_expedients_wizard_views.xml deleted in HEAD and renamed to sipreco_subsidy_management/wizards/create_administrative_process_wizard_views.xml in ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction). Version ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction) of sipreco_subsidy_management/wizards/create_administrative_process_wizard_views.xml left in tree.
CONFLICT (rename/delete): sipreco_subsidy_management/wizards/create_expedients_wizard.py deleted in HEAD and renamed to sipreco_subsidy_management/wizards/create_administrative_process_wizard.py in ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction). Version ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction) of sipreco_subsidy_management/wizards/create_administrative_process_wizard.py left in tree.
CONFLICT (modify/delete): sipreco_subsidy_management/wizards/__init__.py deleted in HEAD and modified in ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction). Version ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction) of sipreco_subsidy_management/wizards/__init__.py left in tree.
CONFLICT (modify/delete): sipreco_subsidy_management/data/ir_actions_server_data.xml deleted in HEAD and modified in ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction). Version ecb7160 ([FIX] sipreco_subsidy_management:expedient creation restriction) of sipreco_subsidy_management/data/ir_actions_server_data.xml left in tree.
Auto-merging sipreco_subsidy_management/__manifest__.py
CONFLICT (content): Merge conflict in sipreco_subsidy_management/__manifest__.py

```

stderr:
```
13:39:22.343095 git.c:444               trace: built-in: git cherry-pick ecb7160dba8ffe852b52a76d96005b1c5627b869
error: could not apply ecb7160... [FIX] sipreco_subsidy_management:expedient creation restriction
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
